package.json

```diff
@@ -3,16 +3,14 @@
   "version": "14.0.0",
   "description": "Format whitespace in a SQL query to make it more readable",
   "license": "MIT",
-  "main": "dist/sql-formatter.min.cjs",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
   "unpkg": "dist/sql-formatter.min.js",
-  "module": "lib/index.js",
-  "types": "lib/src/index.d.ts",
```

Comment:
Removing the `types` field both here and in `exports` lets TypeScript resolve the correct types by filename.